.pipelines/trigger.yaml

```diff
@@ -0,0 +1,51 @@
+pr:
```

Comment:
What is the plan to ensure that this hooks back into PRs?

Reply:
We just need to add a check like we have for ACN. We'll add that after we've merged the second round of changes.

Reply:
Although...I can comment these out until we reach the next phase.